### ISSUE TITLE
Bug causing equipment tooltip to break

### ISSUE THREAD

**Issue by mcdonnelldean** [mcdonnelldean/eso-pad-merchant#3]
Reported on ESOUI: http://www.esoui.com/downloads/info1616-PadMerchant.html#comments

I'm afraid you're addon has broken the equipped items tooltips. But, I came across this issue on Votan's addons, and he found a solution. I've applied the small change to your addon, and the issue is gone.

Basically in Tooltips.lua update your IntegrateWith function to be like the below (basically added a "local result" to the Wrapper function):

__Lua Code:__
```lua
-- Integrates with the given tooltip panel.
local function IntegrateWith(tooltip)
    local method = "LayoutItem"
    local original = tooltip[method]
    
    -- The wrapper will call the old method first, we 
    -- will add our data at the bottom of the list.
    local function Wrapper(self, itemLink, ...)
        local result = original(self, itemLink, ...)
        AddData(self, itemLink)
        return result
    end
 
    tooltip[method] = Wrapper
end
```

**Comment by mcdonnelldean** [comment]
Need to investigate this too. Could not verify since the server is down. Also need to confirm if result should be returned from `original` or `AddData`.

**Comment by mcdonnelldean** [comment]
Should probably be returning the tooltip from `AddData` https://github.com/mcdonnelldean/eso-pad-merchant/blob/master/PadMerchant/addon/Tooltips.lua#L52

**Comment by mcdonnelldean** [comment]
Update: checked equipped item tooltips and they seem fine. Asked for more info from the person who created the issue. Will do more intensive testing tonight.

**Comment by mcdonnelldean** [comment]
Fixed in #5
